### ISSUE TITLE
Release version 2.1.0

### ISSUE THREAD

**Issue by AA-Turner** [sphinx-doc/sphinx-intl#75]
Due to @rffontenelle's work we can now release v2.1.0.

The changelog (CHANGES.rst) should be updated to reflect the move to the new CLI tool and dropping 3.5.

A

**Comment by rffontenelle** [comment]
Should it be 2.1.0 as it is the latest unreleased version?

**Comment by AA-Turner** [comment]
Yes, thanks

**Comment by rffontenelle** [comment]
Transifex APIv2 sunsets in November now and I don't know it is available until Nov 1st or Nov 30, but this issue is now fixed and unreleased. So if I had a vote on this, I would vote to release this as soon as possible to avoid the "Transifex-users world" to reach sphinx-intl's issues because they are unable to use Transifex-related commands. :)

**Comment by rffontenelle** [comment]
Just a reminder that Transifex APIv2 sunsets in 4 days.

FWIW, while 2.1.0 is not released, one can use sphinx-intl with the new transifex CLI tool using the following pip install target: ```git+https://github.com/sphinx-doc/sphinx-intl@6b3913fea5a8c9b819818b9b28b6d0d0b6f022b3```

**Comment by shimizukawa** [comment]
I have received the following email from transifex

```
Given the time-sensitive nature of migrating to the new API, we will still process your requests to the old API for another two weeks, until 12 February 2023. After 12 February, any request made to the old API will be denied.
```

So I think it is a good idea to release sphinx-intl 2.1.0 with support for the new `tx` commands as soon as possible.
Looking at tickets #62, #70, #71, #74 and #77, I believe the work required for the transifex-cli switchover is complete.

I'll do the release work after I merge #81, which drops Python-3.6 with EOL, tomorrow.

@AA-Turner @tk0miya please comment if you have any issues.
Thanks.

**Comment by shimizukawa** [comment]
Now released 2.1.0.
Thanks to all contributors and collaborators！
https://pypi.org/project/sphinx-intl/2.1.0/
